model.go

```diff
@@ -350,17 +346,17 @@ func generateModelType(f *jen.File, m contentfulModel) {
 	)
 
 	var codes = []jen.Code{}
-	var attrs = map[jen.Code]jen.Code{}
+	var attrs = jen.Dict{}
 	generateModelLinkResolver(m, "items", "includes", "cache")(attrs)
 	for k, v := range attrs {
-		codes = append(codes, jen.Sel(jen.Id("tmp"), jen.Add(k)).Op("=").Add(v).Op(";"))
+		codes = append(codes, jen.Id("tmp").Op(".").Add(k).Op("=").Add(v).Op(";"))
```

Comment:
Dot takes a string as the parameter, not a Code, and your k is a Code... Looking at the code, there's probably some easy refactoring that you could do to yield a map[string]Code rather than a Dict. I don't have time to look into it now but it shouldn't be too difficult...